Nodejs/Product/TestAdapter/TestFrameworks/TestFramework.cs

```diff
@@ -35,9 +35,15 @@ public TestFramework(string vsixScriptFolder)
         {
             var testInfo = string.Empty;
             var discoverResultFile = Path.GetTempFileName();
+
+            // Create a temp file with the list of test files.
+            var testFilesString = string.Join(";", testFiles);
+            var testFilesListFile = Path.GetTempFileName();
+            File.AppendAllText(testFilesListFile, testFilesString, System.Text.Encoding.UTF8);
```

Comment:
I would probably `WriteAllText`.  I realize the file should be empty, but I think it's a clearer way to express that you intend to completely determine its contents.

Reply:
Changed.